### DISCUSSION TITLE
Why Windows not boot from OC & Windows partition not shown in macOS?

### DISCUSSION THREAD

**Discussion by omarhaya** [badstorm/surface-pro-7-opencore#56] [Q&A]
hi , Mac OS monetary can't detect windows partition inside Mac OS and I can't boot to windows from open core loader  any one can help?

**Comment by backdash756** [comment]
Hi, mostly this is due to an installed NTFS driver like Paragon NTFS or Tuxera NTFS.  You need to temporarily disable the NTFS driver in order for the Windows partition to show up as a boot volume.  this is not a hackintosh problem and behaves the same on real macs
Have a good Day 👍

**Comment by omarhaya** [comment]
> Hi, mostly this is due to an installed NTFS driver like Paragon NTFS or Tuxera NTFS. You need to temporarily disable the NTFS driver in order for the Windows partition to show up as a boot volume. this is not a hackintosh problem and behaves the same on real macs
> Have a good Day 👍

windows boot drive is shown on the open core bootloader menu but when I boot from it I get recovery error but when I choose windows boot option from bios windows 11 boots normally

**Comment by czlzd** [comment]
repair with windowsPE，than copy the efi to your efi folder

**Comment by backdash756** [comment]
> > Hi, mostly this is due to an installed NTFS driver like Paragon NTFS or Tuxera NTFS. You need to temporarily disable the NTFS driver in order for the Windows partition to show up as a boot volume. this is not a hackintosh problem and behaves the same on real macs
> > Have a good Day 👍
> 
> windows boot drive is shown on the open core bootloader menu but when I boot from it I get recovery error but when I choose windows boot option from bios windows 11 boots normally

I misunderstood the situation, sorry

**Comment by badstorm** [comment]
I think you installed Windows with Secure Boot enabled. If you want to boot Windows from OC you must install it witn Secure Boot disabled.

Since this request is not related to che OC configuration i'll move it to discussion.
